easybuild/easyconfigs/m/Mesa/Mesa-12.0.2-intel-2016b.eb

```diff
@@ -8,13 +8,14 @@
 easyblock = 'ConfigureMake'
 
 name = 'Mesa'
-version = '12.0.1'
+version = '12.0.2'
 
 homepage = 'http://www.mesa3d.org/'
 description = """Mesa is an open-source implementation of the OpenGL specification -
  a system for rendering interactive 3D graphics."""
 
 toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'optarch': False}
```

Comment:
@wpoely86 please add a comment to clarify why you're doing this

other than that: good to go!